indico/modules/events/contributions/lists.py

```diff
@@ -122,7 +122,7 @@ def render_list(self, contrib=None):
         :param contrib: Used in RHs responsible for CRUD operations on a
                         contribution.
         :return: dict containing the list's entries, the fragment of
-                 displayed entries and whether the contrib passed is displayed
+                 displayed entries and whether the contribution passed is displayed
```

Comment:
:disappointed: http://stackoverflow.com/questions/24125957/multiple-right-margins-in-pycharm
I'll take it into account tho